Core/Tests/Speckle.Core.Tests.Integration/Api/GraphQL/Resources/ActiveUserResourceTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Speckle.Core.Api;
```

Comment:
Probably we are happily newing some things up and the test are okay with this, this is probably okay, probably the qraphql class we create in the graphql ops is lighweight and less of an issue

Reply:
Not sure I'm quite following what you're saying here Ian, could you clarify?

Reply:
This looks like it's on the wrong line, I was referring to using new within some of the methods and wondering if units tests would be a problem but later I see you added a bunch